plugin/api.go

```diff
@@ -74,6 +74,11 @@ type API interface {
 	// Minimum server version: 5.6
 	SetTeamIcon(teamId string, data []byte) *model.AppError
 
+	// RemoveTeamIcon removes the Team Icon.
```

Comment:
```suggestion
	// RemoveTeamIcon removes the team icon.
```

Reply:
done

Reply:
I propose we also fix the other occurrences of this Term in https://github.com/mattermost/mattermost-server/blob/78cc31430fde7ed050fb0bba9a609f2c384445b5/plugin/api.go#L67 and
https://github.com/mattermost/mattermost-server/blob/78cc31430fde7ed050fb0bba9a609f2c384445b5/plugin/api.go#L72

Reply:
done @hanzei